oak_containers_stage1/src/dice.rs

```diff
@@ -143,24 +64,10 @@ struct MemoryRange {
 
 /// Extracts the DICE evidence and ECA key from the Stage 0 DICE data located at the given physical
 /// address.
-///
-/// Also zeroes out the original copy of the signing key.
-pub fn extract_stage0_evidence_and_key(start: PhysAddr) -> anyhow::Result<DiceDataBuilder> {
-    let mut dice_data = read_stage0_dice_data(start)?;
-    let result = DiceDataBuilder::new(
-        dice_data_to_proto(&dice_data)?,
-        SigningKey::from_slice(
-            &dice_data.layer_1_certificate_authority.eca_private_key[..P256_PRIVATE_KEY_SIZE],
-        )?,
-    );
-
-    // Clear the private key bytes.
-    dice_data
-        .layer_1_certificate_authority
-        .eca_private_key
-        .copy_from_slice(&EMPTY_KEY);
-
-    Ok(result)
+pub fn extract_stage0_dice_data_builder(start: PhysAddr) -> anyhow::Result<DiceDataBuilder> {
```

Comment:
Should it be `extract_stage0_into_dice_data_builder`?

Reply:
I decided to go with `extract_stage0_dice_data`, since the result type is already obvious from the function signature.